sympy/functions/elementary/complexes.py

```diff
@@ -654,6 +710,10 @@ class arg(Function):
     pi/2
     >>> arg(sqrt(2) + I*sqrt(2))
     pi/4
+    >>> arg(sqrt(3)/2 + I/2)
+    pi/6
+    >>> arg(4/5 + 3*I/5)
+    0.643501108793284
```

Comment:
This example can be confusing because it implicitly ignores how `4/5` can turn out to be floating point coefficient than rational.

Reply:
On the sympy live shell, running this gives atan(3/4). But while running tests, the output is the floating value. Any reason why this is happenning?

Reply:
Sympy live can use some hooks to comprehend 3/4 as rationals than floats

Reply:
I got it. arg(4 + 3*I) gives atan(3/4). arg(4/5 + 3*I/5) is evaluated before being passed, so it returns float.

Reply:
Let's avoid examples that use floating point numbers, especially in cases like this where the numbers are not exact.